### DISCUSSION TITLE
Slippage on live trading?

### DISCUSSION THREAD

**Discussion by sfolaxsel** [conor19w/Binance-Futures-Trading-Bot#42] [Q&A]
I want to apply the slippage used in the backtester to live trading. (Hardcoding is also fine.) 
Which function should I apply it to? 
Helper.py's open_trade_check_threshold?
And what role does the trading threshold play?

**Comment by conor19w** [comment]
Not sure what you mean, slippage in the backtester is a simulated slippage. In practice its hard to know how much slippage you will experience but you don't need to account for it additionally as binance handles order execution for you.
Trading threshold is something you can set to avoid trades that have moved the direction you wanted before you got a chance to enter the position, trades will be cancelled so they arent later entered when price is coming back towards your order
